build/Publish.targets

```diff
@@ -175,15 +175,23 @@
                     AccountKey="$(PublishBlobFeedKey)"
                     ItemsToPush="@(PackagesToPublishToTransport)"
                     Overwrite="$(PushToBlobFeed_Overwrite)"
+                    ManifestBuildId="$(BuildNumber)"
+                    ManifestCommit="$(CommitHash)"
+                    ManifestName="https://github.com/aspnet/Home"
                     MaxClients="$(PushToBlobFeed_MaxClients)"
+                    SkipCreateManifest="false"
```

Comment:
Why is this false?

Reply:
I looked at what they had in their other PRs e.g. https://github.com/dotnet/templating/pull/1400/files. Assumed this needed to be set.

Reply:
Ok. I assume we need it then.

cc @dagood should we call this out in https://github.com/dagood/core-eng/blob/orchestration/Documentation/Orchestrated-Build/Api/publish.md?

Reply:
It's [optional and defaults to `false`](https://github.com/dotnet/buildtools/blob/cd0812d52c432a415fb625bca5dd790adc8136a9/src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs#L44), no need for this line. :smile: The core\* repos don't include it.

Reply:
That said, I'll note it in the doc so the property isn't a mystery.